eisbuk-admin/src/components/atoms/AttendanceCard/AttendanceCard.tsx

```diff
@@ -54,6 +54,8 @@ const AttendanceCard: React.FC<Props> = ({
 
   const timeString = `${startTime} - ${endTime}`;
 
+  /** @TODO remove this when slots have intervals */
+  const dummyIntervals = ["13:00 - 14:00", "13:15 - 14:15"];
```

Comment:
This should be received as props (so no need for `dummyIntervals`. If you need preview, that's exactly what storybook is for.

The props should look like this
```typescript
interface Props {
  ...propsYouHaveSoFar
  attendance: { // instead of absentees
    [customerId]: {
      bookedInterval: string | null
      attendedInterval: string | null  
    }
  }
  intervals: {
    [key: string]: {  // i.e. "08:00-09:30", don't ask why...we're doing it like this for now, but I'm rethinking it every day
      startTime: string // i.e. 08:00
      endTime: string // i.e. 09:30
    }
  }
}
```

Reply:
shouldn't `intervals` be a prop for `UserAttendance` ?
also `customerId ` is type string right?

Reply:
You can use `SlotInterface` from `@/types/temp`. I've created that file to contain the new data model until it's intergrated.

Reply:
So...you just need to extend `SlotInterface` instead of `Slot<"id">`